monai/apps/reconstruction/networks/nets/utils.py

```diff
@@ -131,3 +134,43 @@ def complex_normalize(x: Tensor) -> Sequence:  # type: ignore
         )
         x = x.view(b, c, h, w, d)
         return (x - mean) / std, mean, std  # type: ignore
+
+
+def sens_reduce(x: Tensor, sens_maps: Tensor, spatial_dims: int = 2) -> Tensor:
+    """
```

Comment:
Could you help spell out the func name? Is it `reduce_sensitivity_map`?

Reply:
Or maybe `sum_sensitivity_map`?

Reply:
If x is kspace signal, would it be better to call it kspace signal? I did not remember how you name kspace signal in your previous PRs.

Reply:
yeah sure, maybe `sensitivity_map_reduce` and `sensitivity_map_expand`... The terms `reduce` and `expand` have intuitive counter meanings.

Reply:
yeah kspace sounds good